src/marqo/tensor_search/api.py

```diff
@@ -83,8 +83,8 @@ def marqo_base_exception_handler(request: Request, exc: base_exceptions.MarqoErr
         # Vespa client exceptions
         (
             vespa_exceptions.VespaTimeoutError,
-            api_exceptions.TooManyRequestsError,
-            "Throttled by vector store. Try your request again later."
+            api_exceptions.VectorStoreTimeoutError,
+            "Vector store request timed out"
```

Comment:
maybe add "try again later" to the error message (hint at the best way to deal with this error client-side)